ticker.py

```diff
@@ -0,0 +1,135 @@
+from time import time
+import json
+import toml
+import requests
+import argparse
+from urllib import urlencode
+
+PAGE_LIMIT = 200  # for aggregation endpoint
+
+
+def millis():
+    """current time in millis since epoch"""
+    return int(round(time() * 1000))
+
+
+def get_asset_params(prefix, atype, code, issuer):
+    """get aggregation request parameters for single asset"""
+    return {
+        prefix + "_asset_type": atype,
+        prefix + "_asset_code": code,
+        prefix + "_asset_issuer": issuer,
+    }
+
+
+def get_asset_param_from_pair(pair, prefix):
+    """get aggregation parameters for asset pair"""
+    if pair[prefix + "_asset_issuer"] == "native":
+        return get_asset_params(prefix, "native", "", "")
+    else:
+        return get_asset_params(prefix, "credit_alphanum4", pair[prefix + "_asset_code"],
+                                pair[prefix + "_asset_issuer"])
+
+
+def get_aggregation_params(pair, start, end, resolution):
```

Comment:
here too `make_*` since it's really creating a new object rather than  fetching values